tests/behaviour/background/environment_base.py

```diff
@@ -78,6 +80,9 @@ def after_scenario(context: Context, scenario):
     if scenario.status == Status.skipped:
         return
 
+    #TODO: REMOVE THIS ONCE THE CRASHES ARE FIXED
+    time.sleep(1)
```

Comment:
`after_scenario` is literally run after every single scenario. Given that each Graql language file has around 60-140 scenarios, and we run Core and Cluster versions of each feature, this will easily add 4 minutes to the overall runtime in CI.

We fixed a similar issue in Cluster by adding a 0.01 second sleep. I would propose we do the same here.
```suggestion
    time.sleep(0.01)
```

Reply:
Modified.